src/Service.Tests/SqlTests/GraphQLPaginationTests/MsSqlGraphQLPaginationTests.cs

```diff
@@ -43,7 +43,7 @@ public static async Task SetupAsync(TestContext context)
             DisplayName = "Test after token for decimal values.")]
         [DataRow("boolean_types", "false", "true", 2, 4,
             DisplayName = "Test after token for boolean values.")]
-        [DataRow("date_types", "\"0001-01-01\"",
+        [DataRow("date_types", "\"1753-01-01\"",
```

Comment:
why this change? The range of SQL Date data type is from January 1, 0001, through December 31, 9999.

Reply:
Yeah, corrected. Thanks!